spec/services/clients/datacite_spec.rb

```diff
@@ -21,7 +21,7 @@
       let(:results) { client.list(client_id: 'sul.openneuro', page_size: 100) }
 
       it 'retrieves the list of datasets' do
-        expect(results.size).to eq(4385)
+        expect(results.size).to eq(4418)
```

Comment:
I cleared out the datacite cassettes when testing, so these numbers got bumped.

Reply:
cool!